### DISCUSSION TITLE
Include a table of combos achieved during play and related statistics to results screen

### DISCUSSION THREAD

**Discussion by farroxus** [ppy/osu#13181] [Ideas]
conceptualization image-  
![opera_2021-02-23_18-43-53](https://user-images.githubusercontent.com/79548888/108923588-1723c180-7607-11eb-9718-bd514fb46644.png)

In plays of a map where combo was broken at least once, include a simple table which conveys the appropriate information for analyzation, either immediately visible and prominent, or hidden and revealed upon clicking some button/object 

I don't really understand how this works, sorry if this isn't correctly formatted or I don't have enough information. Just a simple feature request.

**Comment by peppy** [comment]
Sounds like an interesting concept, though it should probably be sorted / limited to top combos only? If a user was to break combo 100 times we probably don't want this table to show 100 rows.

**Comment by smoogipoo** [comment]
I think a more useful way to label this is by kiai time, and it's generally what higher-rated players will use when referring to difficulty strains in a map. Most maps will have 1-2 kiais, maybe 3-4 for longer (3-4 minute) maps, leading to at most 8 rows (non-kiai + kiai).

You could also probably merge the non-kiai rows as they're generally pretty similar sections, at which point that could further be condensed to 5 rows.
